docs/sources/panels-visualizations/panel-overview/index.md

```diff
@@ -0,0 +1,161 @@
+  - **Copy**: Copy the panel to the clipboard.
+  - **Create library panel**: Create a panel that can be imported into other dashboards.
+  - **Create alert**: Open the alert rule configuration page in **Alerting**, where you can [create a Grafana-managed alert][create] based on the panel queries.
+  - **Hide legend**: Hide the panel legend.
+  - **Get help**: Send a snapshot or panel data to Grafana Labs Technical Support.
+- **Remove**: Remove the panel from the dashboard.
+
+## Keyboard shortcuts
+
+Grafana has a number of keyboard shortcuts available specifically for panels. Press `?` on your keyboard to display all keyboard shortcuts available in your version of Grafana.
+
+By hovering over a panel with the mouse you can use some shortcuts that will target that panel.
+
+- `e`: Toggle panel edit view
+- `v`: Toggle panel fullscreen view
+- `ps`: Open Panel Share Modal
+- `pd`: Duplicate Panel
+- `pr`: Remove Panel
+- `pl`: Toggle panel legend
+
+## Add a panel
+
+To add a panel in a new dashboard click **+ Add visualization** in the middle of the dashboard:
+
+![Empty dashboard state](/media/docs/grafana/dashboards/empty-dashboard-10.2.png)
+
+To add a panel to an existing dashboard, click **Add** in the dashboard header and select **Visualization** in the drop-down:
+
+![Add dropdown](/media/docs/grafana/dashboards/screenshot-add-dropdown-10.0.png)
+
+## Panel configuration
+
+To configure panels, refer to the following subtopics:
+
+- [Configure panel options][]
+- [Configure standard options][]
+- [Configure a legend][legend]
+- [Configure tooltips][tooltips]
+- [Configure data links][]
+- [Configure value mappings][]
+- [Configure thresholds][]
+- [Configure field overrides][]
+
+{{% docs/reference %}}
+[query]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data"
+[query]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data"
+
+[visualization]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/visualizations"
+[visualization]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/visualizations"
+
+[transformations]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/panels-visualizations/query-transform-data/transform-data"
+[transformations]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/visualizations/panels-visualizations/query-transform-data/transform-data"
+
+[Data source management]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/administration/data-source-management"
+[Data source management]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/administration/data-source-management"
+
+[Data sources]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/datasources"
+[Data sources]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA VERSION>/datasources"
```

Comment:
For consistency (and because space separated is older, deprecated syntax).

Also I think we can link to the data sources documentation in Grafana Cloud now (https://grafana.com/docs/grafana-cloud/connect-externally-hosted/data-sources/)

```suggestion
[Data source management]: "/docs/grafana/ -> /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management"
[Data source management]: "/docs/grafana-cloud/ -> /docs/grafana/<GRAFANA_VERSION>/administration/data-source-management"

[Data sources]: "/docs/grafana/ -> /docs/grafana/<GRAFANA VERSION>/datasources"
[Data sources]: "/docs/grafana-cloud/ -> /docs/grafana-cloud/connect-externally-hosted/data-sources"
```